src/editors/containers/VideoEditor/components/VideoSettingsModal/components/VideoSourceWidget/hooks.jsx

```diff
@@ -0,0 +1,17 @@
+import { actions } from '../../../../../../data/redux';
+
+// export const state = {
+//   onFocusVideoId: (args) => React.useState(args),
+//   onFocus
+// };
+export const deleteFallbackVideo = ({ fallbackVideos, dispatch }) => (videoUrl) => {
```

Comment:
Because this is working off of the videoUrl string, it is deleting all Fallback Video fields with the same string when you click delete on any of the fields (including empty fields). A nodelist is a structure that comes to mind for this.